docs/README.md

```diff
@@ -23,14 +23,25 @@ launch
     or "Greetings, human."
   say "What's your name?"
 
-  when "My name is $name"
+  when "(my|the) name is $name"
+    or "(you can|you should|) call me $name"
```

Comment:
nit: is the `|` after "should" intentional?

Reply:
Yup, the last alternation is specifically allowed to be empty to denote, "or nothing".